### PR TITLE
IDE-903 Crash on file deletion in repository tree

### PR DESCRIPTION
**Pull request by dehilsterlexis**
Was missing logic for user-marked deletions.

### CODE REVIEW

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by dehilsterlexis** [comment]
@richardkchapman hold off. I am checking to see if I pushed the wrong version.

**Comment by dehilsterlexis** [comment]
@GordonSmith for review. I forgot to add the SourceView files to the commit. That is the only change from before.

File: wlib/SourceView.h [hunk 1]
```diff
@@ -183,6 +183,7 @@ class CSourceCtrl : public CWindowImpl<CSourceCtrl, CScintillaCtrlEx, CScintilla
 
     int GetText(CString & pstrText) const;
     bool IsTextSelected();
+	IAttribute *GetAttribute();
```
Comment (GordonSmith) [inline wlib/SourceView.h]: Indentation

**Comment by dehilsterlexis** [comment]
@GordonSmith for review

**Comment by GordonSmith** [comment]
@richardkchapman looks good to go.

**Pull request merged**